include/deal.II/grid/grid_tools.h

```diff
@@ -3290,8 +3216,7 @@ namespace GridTools
                         tria->get_communicator(), &status);
         AssertThrowMPI(ierr);
 
-        CellDataTransferBuffer<dim, DataType> cellinfo;
-        cellinfo.unpack_data(receive);
+        auto cellinfo = Utilities::unpack<CellDataTransferBuffer<dim, DataType> >(receive);
```

Comment:
out of curiosity, how are `std::vector<CellId> cell_ids;` (member of `CellDataTransferBuffer`) being set here?

Reply:
p.s. I am just a bit lost how do we unpack this part of the object here... @bangerth ?

Reply:
I have no idea...

Reply:
ah, i guess it's hidden in `archive << object;` and `archive >> object;` which calls `load` and `save` methods of this class and consequently do the right thing.